src/test/resources/htsjdk/variant/test_withNanQual.vcf

```diff
@@ -0,0 +1,8 @@
+##fileformat=VCFv4.1
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##contig=<ID=chr1,length=14640000>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878
+chr1	1	.	C	T	NaN	.	.	GT:GQ	./.:NaN
+chr1	2	.	C	T	nan	.	.	GT:GQ	./.:nan
+chr1	3	.	C	T	-nan	.	.	GT:GQ	./.:-nan
+chr1	4	.	C	T	+nan	.	.	GT:GQ	./.:+nan
```

Comment:
While were' at it, please also test inf and infinity and the various capitalizations....